src/humancellatlas/data/metadata/__init__.py

```diff
@@ -155,7 +155,7 @@ class DonorOrganism(Biomaterial):
     disease: Set[str]
     organism_age: str
     organism_age_unit: str
-    biological_sex: str
+    sex: Optional[str]
```

Comment:
`sex` is required: https://github.com/HumanCellAtlas/metadata-schema/blob/integration/json_schema/type/biomaterial/donor_organism.json#L5

Reply:
Sorry, I'm confused a little bit if that's the case, should be also change https://github.com/HumanCellAtlas/metadata-api/blob/1b7192cecbef43b5befecc4153bf2e2f4db5bb16/src/humancellatlas/data/metadata/__init__.py#L110 since `project_short_name` is mandatory: https://github.com/HumanCellAtlas/metadata-schema/blob/d81b2f59f86cd40015a10112ea247d3b71f048b9/json_schema/core/project/project_core.json#L6 ?

Reply:
@hannes-ucsc ^